src/Angor/Client/Pages/Settings.razor

```diff
@@ -906,6 +911,10 @@
             notificationComponent.ShowErrorMessage("Failed to wipe data: " + e.Message);
         }
     }
+    private void ChangeFeatureFlags()
```

Comment:
if this method is called only once then I rather delete it and put the code in the `ChangeNetwork` method